Classes/Module/Statistics.php

```diff
@@ -1850,4 +2016,21 @@ protected function getLanguageService()
     {
         return $GLOBALS['LANG'];
     }
+
+
```

Comment:
please add some PHPDoc for this function

Reply:
This function was already present in the original code, it's not our code change